src/hooks/useAutoSlippageTolerance.ts

```diff
@@ -61,8 +75,8 @@ export default function useAutoSlippageTolerance(
     // if not, use local heuristic
     const dollarCostToUse =
       chainId && SUPPORTED_GAS_ESTIMATE_CHAIN_IDS.includes(chainId) && trade?.gasUseEstimateUSD
-        ? trade.gasUseEstimateUSD
-        : dollarGasCost
+        ? trade.gasUseEstimateUSD.multiply(GAS_ESTIMATE_BUFFER)
```

Comment:
not sure i understand why you're multiplying by 10% here?

Reply:
Yeah its all pretty rough but my thinking is:

1/ The gas estimates from the routing API and from the guesstimate function will over-estimate in some cases and under-estimate in other cases. 

2/ Underestimates are bad as we end up setting slippage ultra tight which leads to transaction failures => bad ux

3/ On Arbitrum transaction failures rates are hovering around 30% which feels pretty high https://dune.com/queries/565180/1059058 

4/ This whole auto slippage stuff is ultra rough anyway and adding 10% here or there doesn't really mean much in the grand scheme. It should lean us slightly more towards better UX (less failures) with a very hard to quantify con of user getting a worse trade due to being front run or slippage

Reply:
that is polygon not arbitrum btw

Reply:
I'll kindly reask the question. From the explanation it is still not clear why is it multiplying by 10% instead of adding extra 10%?